examples/vision/object_recognition.py

```diff
@@ -16,20 +16,30 @@
 from PIL import Image
 
 from camel.agents import ChatAgent
+from camel.configs import ChatGPTConfig
 from camel.generators import PromptTemplateGenerator
 from camel.messages import BaseMessage
-from camel.types import ModelType, RoleType, TaskType
+from camel.models import ModelFactory
+from camel.types import (
+    ModelPlatformType,
+    ModelType,
+    RoleType,
+    TaskType,
+)
 
 parser = argparse.ArgumentParser(description="Arguments for object detection.")
 parser.add_argument(
-    "--image_path",
+    "--image_paths",
```

Comment:
Now seems that object recognition only supports one image. `image_path` should be ok

Reply:
It supports multiple images recongnition.